source/lex.tex

```diff
@@ -525,6 +525,12 @@
 characters \tcode{//} and \tcode{/*} have no special meaning within a
 \tcode{/*} comment.
 \end{note}
+\begin{note}
+Comments are turned into \unicode{0020}{space} characters in
+phase 3 of translation as part of decomposing a source file into
+preprocessing tokens and sequences of whitespace characters.
```

Comment:
lex.phases p3 says a bit much in that location.

Maybe it's better to move the normative provision "Each comment (5.4) is replaced by one u+0020 space character." here or into lex.pptoken, possibly as part of the "whitespace grammar" paper.

Reply:
I was reluctant to touch the phases of translation more than necessary in P3657, the whitespace grammar paper.  However, if we prefer that direction I will updated the next revision of that paper and close this PR.